### PR TITLE
configurable notebook location

### LINKED ISSUE #247: Polynote's notebook directory should be configurable

### PR DESCRIPTION
**Pull request by indig**
Resolves #247

### CODE REVIEW

**Comment by jeremyrsmith** [comment]
Looks like this causes a build failure in `PolynoteConfigSpec`... I know there's a lot of spurious travis failures but this looks legit.

File: polynote-server/src/main/scala/polynote/server/Server.scala [hunk 1]
```diff
@@ -38,19 +38,19 @@ trait Server extends IOApp with Http4sDsl[IO] with KernelLaunching {
     }
   }
 
-  def downloadFile(path: String, req: Request[IO]): IO[Response[IO]] = {
-    val nbLoc = new File(System.getProperty("user.dir")).toPath.resolve(s"notebooks/$path").toString
+  def downloadFile(path: String, req: Request[IO], config: PolynoteConfig): IO[Response[IO]] = {
```
Comment (indig) [inline polynote-server/src/main/scala/polynote/server/Server.scala]: Having to thread the config all the way through here is a little annoying...

**Comment by indig** [comment]
My bad! fixing...

**Comment by indig** [comment]
Fixed!

**Pull request merged**